railties/lib/rails/application/configuration.rb

```diff
@@ -119,6 +119,7 @@ def load_defaults(target_version)
 
           if respond_to?(:action_view)
             action_view.default_enforce_utf8 = false
+            action_view.button_to_generates_input = false
```

Comment:
Can you move to the 6.1 defaults instead of 6.0?

Reply:
Moved and rebased to the latest master commit.